tests/conftest.py

```diff
@@ -16,13 +29,41 @@
 from lbz.resource import Resource
 from lbz.response import Response
 from lbz.router import Router, add_route
-from tests import SAMPLE_PRIVATE_KEY
+from tests.fixtures.rsa_pair import SAMPLE_PRIVATE_KEY, SAMPLE_PUBLIC_KEY
 from tests.utils import encode_token
 
 
 @pytest.fixture(scope="session", name="allowed_audiences")
 def allowed_audiences_fixture() -> List[str]:
-    return os.environ["ALLOWED_AUDIENCES"].split(",")
+    return [str(uuid4()), str(uuid4())]
+
+
+@pytest.fixture(autouse=True, name="patch_environ")
+def patch_environ_fixture(allowed_audiences: List[str]) -> Iterator[None]:
+    with patch.dict(
+        environ,
+        {
+            "AUTH_REMOVE_PREFIXES": "1",
+            "ALLOWED_PUBLIC_KEYS": json.dumps({"keys": [SAMPLE_PUBLIC_KEY]}),
+            "ALLOWED_AUDIENCES": ",".join(allowed_audiences),
+            "ALLOWED_ISS": "test-issuer",
+            "AWS_LAMBDA_FUNCTION_NAME": "million-dollar-lambda",
+            "EVENTS_BUS_NAME": "million-dollar-lambda-event-bus",
+            "AWS_DEFAULT_REGION": "us-west-2",
+        },
+        clear=True,
+    ):
```

Comment:
As I mentioned in the past, avoid creating multiline context managers - you can always declare a variable above 😉